pkg/util/utils.go

```diff
@@ -652,3 +660,120 @@ func ValidateSysctls(strSlice []string) (map[string]string, error) {
 	}
 	return sysctl, nil
 }
+
+func DefaultContainerConfig() *config.Config {
```

Comment:
Please document/comment exposed types and APIs. Currently it's passing CI because we (very unfortunately) have `golint` disabled but we need to get that back. Documenting it now will spare us some work in the future.

Reply:
Moving these to containers.conf